LibTrixi.jl/src/LibTrixi.jl

```diff
@@ -31,12 +31,24 @@ export trixi_nelements,
 export trixi_nelements_global,
        trixi_nelements_global_cfptr,
        trixi_nelements_global_jl
+export trixi_ndofs,
+       trixi_ndofs_cfptr,
+       trixi_ndofs_jl
+export trixi_ndofs_global,
+       trixi_ndofs_global_cfptr,
+       trixi_ndofs_global_jl
+export trixi_ndofs_element,
+       trixi_ndofs_element_cfptr,
+       trixi_ndofs_element_jl
```

Comment:
Below, we have `trixi_load_cell_averages` and friends. I think it would be prudent to decide whether to call it `elements` or `cells` in the context of libtrixi, similar to what we did for Trixi.jl.

If you decide to not make a distinction between cells and elements in libtrixi, it might make things easier for libtrixi users, but harder for Trixi.jl users. However, in that case I recommend to pick one (either "cells" or "elements") and use it consistently everywhere, but to not mix and match. If you want to continue to use both like in Trixi.jl, I recommend changing "trixi_load_cell_averages" accordingly (not in this PR, but in a future one)

Reply:
Note that this distinction (or not) should also be consequently used in all comments, where, I believe, also cells and elements are used currently.